schema/deploy/warehouse/sample.sql

```diff
@@ -0,0 +1,34 @@
+-- Deploy seattleflu/schema:warehouse/sample to pg
+-- requires: warehouse/encounter
+
+begin;
+
+set search_path to warehouse;
+
+create table sample (
+    sample_id integer primary key generated by default as identity,
+    identifier text unique not null,
+    encounter_id integer, -- references encounter (encounter_id), not null,
+    details jsonb
+);
+
+-- We removed the encounter_id reference to the encounter table because this relationship
+--   does not exist yet in the database. We will add the relationship back in once
+--   Thomas has finished with the barcode ingestion part of the pipeline.
+--   At that time, we can also decide whether or not to add a `not null` constraint to 
+--   encounter_id. 
+-- We are tracking some part of the JSON document in a column for additional details. 
+--   We may or may not find this column useful down the road. 
+-- We also need to consider whether `encounter_id` should accept null values. This will
+--   depend on how we choose to handle the clinic data (data without collection ids).
+
+comment on table sample is 'A sample collected from an individual during a specific encounter';
+comment on column sample.sample_id is 'Internal id of this sample';
+comment on column sample.identifier is 'A UUID assigned to this sample';
```

Comment:
It's a unique external identifier, at least, but not always a UUID. Maybe update the comment here?